recipes.mdwn

```diff
@@ -8,6 +8,7 @@ to improve your Awesome setup.
 ## Setups
 
 * [My First Awesome Setup](https://awesomewm.org/apidoc/documentation/07-my-first-awesome.md.html#)
+* [A step by step Guide](https://epsi-rns.github.io/desktop/2019/06/15/awesome-overview.html) (for Awesome 4.3)
```

Comment:
```suggestion
* [A step by step Guide by Epsi](https://epsi-rns.github.io/desktop/2019/06/15/awesome-overview.html) (for Awesome 4.3)
```

since the amount of articles in his cycle about awesome wm customization is way too big for being properly reviewed, i think it's important to add such sort of mention what it's not an official guide

Reply:
Aha... Thank you.

Reply:
Sorry for my english.
What am I supposed to do?
What should I do here to contribute?

Reply:
nothing, that's a note for author of this PR or for a person who will be merging this